domains/qwertystars.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "qwertystars",
+        "email": "srijan.guchhait@gmail.com"
+    },
+    "record": {
+        "CNAME":"qwertystars-ts-server.serveo.net"
```

Comment:
Can't reach website, Error 502, Could you provide a preview of your website?

Reply:
should be online i again. (unfortunately the current method of making my site available, is well buggy). looking for a more stable way. will request a review later (2-3 months) when site functionality is stable.